near-sdk-macros/src/core_impl/code_generator/item_impl_info.rs

```diff
@@ -23,13 +52,41 @@ impl ItemImplInfo {
             Err(e) => syn::Error::new(self.ty.span(), e).to_compile_error(),
         }
     }
+
+    pub fn generate_error_methods(&self) -> TokenStream2 {
+        let mut error_methods = quote! {};
+
+        self.methods.iter().map(|m| &m.attr_signature_info).for_each(|method| {
```

Comment:
Let's add a warning that _error suffix in method identifier is reserved for our usage